ExampleMod/ExampleMod.cs

```diff
@@ -13,6 +13,7 @@ public partial class ExampleMod : Mod
 
 		public override void Load() {
 			// Registers a new custom currency
+			CloudLoader.AddCloudFromTexture<SimpleModCloud>(this, AssetPath + "Textures/CloudsT/SimpleCloud");
```

Comment:
Move this down so the comment above still comments the ExampleCustomCurrencyId  line